Before: 603c76a37c963eeab6022ea176217f7e376de26219eb87663264982b9d68c6fc
After: be4e4e02e6bece079c79d9ab595e8e9452968b1fc5f2bc444423e6ff105b43ef
Patch: 2685062b29f5caa08f1235cb400358a2013fac9cde1a40a93c52b29de18b8aee
--- lib/corvid/%name%_plugin.rb.tt	2012-08-14 11:33:00.238579238 +1000
+++ lib/corvid/%name%_plugin.rb.tt	2012-08-14 11:44:42.120003135 +1000
@@ -5,11 +5,6 @@
   # Plugins can be extensions but are not by default.
   #include Corvid::Extension
 
-  # The name of the plugin. Must conform to format enforced by {Corvid::NamingPolicy#validate_plugin_name!}.
-  #
-  # @return [String] The plugin name.
-  name '<%= name %>'
-
   # The path for Ruby to require in order to load this plugin.
   #
   # @return [String] The path to require, usually relative to your lib dir.
